flexmeasures/utils/coding_utils.py

```diff
@@ -165,3 +168,13 @@ def find_classes_modules(module, superclass, skiptest=True):
 
 def get_classes_module(module, superclass, skiptest=True) -> dict:
     return dict(find_classes_modules(module, superclass, skiptest=skiptest))
+
+
+def validate_color_hex(ctx, param, value):
```

Comment:
Because this is really tied to be used by click as option callback, I would not out this here, but in `cli/utils.py`

Reply:
ok, it can be moved

Reply:
I also think this should be there now that i think about it, as its related to cli, still gettgin used to teh structure of the project.